tests/stubtest_third_party.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+"""Test typeshed's third party stubs using stubtest"""
+
+import subprocess
+import sys
+import tempfile
+import toml
+import venv
+from pathlib import Path
+
+
+EXCLUDE_LIST = ["Flask", "pyaudio", "backports", "pkg_resources", "six"]
+
+
+class StubtestFailed(Exception):
+    ...
+
+
+def run_stubtest(dist: Path) -> None:
+    with open(dist / "METADATA.toml") as f:
+        metadata = dict(toml.loads(f.read()))
+
+    # Ignore stubs that don't support Python 2
+    if not bool(metadata.get("python3", True)):
+        return
+
+    with tempfile.TemporaryDirectory() as tmp:
+        venv_dir = Path(tmp)
+        venv.create(venv_dir, with_pip=True, clear=True)
+
+        pip_exe = str(venv_dir / "bin" / "pip")
+        python_exe = str(venv_dir / "bin" / "python")
+
+        # We need stubtest to be able to import the package, so install mypy into the venv
+        # Hopefully mypy continues to not need too many dependencies
+        # TODO: Pick package version based on metadata.get("version")
```

Comment:
We should probably clean up all the metadata versions first.

Reply:
I was thinking we could just special case 0.1 versions here to just install pypi's latest.